classes/controller/AdminController.php

```diff
@@ -2856,6 +2856,12 @@ public function initNotifications()
             'show_new_customers' => Configuration::get('PS_SHOW_NEW_CUSTOMERS'),
             'show_new_messages' => Configuration::get('PS_SHOW_NEW_MESSAGES '),
         ));
+
+        Media::addJsDef(array(
+            'show_new_orders' => Configuration::get('PS_SHOW_NEW_ORDERS'),
+            'show_new_customers' => Configuration::get('PS_SHOW_NEW_CUSTOMERS'),
+            'show_new_messages' => Configuration::get('PS_SHOW_NEW_MESSAGES '),
+        ));
```

Comment:
If this change is going to be merged, it would be better to reuse the same array by storing it in a variable.